src/main/java/org/spongepowered/common/data/persistence/ConfigurateTranslator.java

```diff
@@ -75,64 +73,56 @@ private static DataContainer translateFromNode(ConfigurationNode node) {
         return dataContainer;
     }
 
-    private void translate(Map<Object, ? extends ConfigurationNode> node, DataView container, Map<?, ?> value) {
+    private void translate(Map<Object, ? extends ConfigurationNode> node, DataView container, Object key, Map<?, ?> value) {
```

Comment:
The `Object key` parameter is not used here, but is in the list variant.

In both cases, the key and value parameters are always the key and value of the node from what I see. Passing them as parameters is unnecessary.